lib/ansible/modules/cloud/amazon/ec2_asg.py

```diff
@@ -1622,6 +1639,12 @@ def asg_exists(connection):
     return bool(len(as_group))
 
 
+def fail_if_max_instance_lifetime_not_supported():
+    if LooseVersion(boto3.__version__) < LooseVersion('1.10.0'):
```

Comment:
We have a helper for this which means you don't need to explicitly import LooveVersion and boto3:

```suggestion
    if not module.boto3_at_least('1.10.0'):
```

Dumb question, is it actually the boto3 version that's needed or of there a more specific botocore version?

Reply:
Oh,I just realised this module's still AnsibleModule rather than AnsibleAWSModule.

#66863 might be the better version here since it includes some other cleanup.

Reply:
@tremble - I have limited my patch to just fixing #66716. I tried to keep the changes for this PR to the minimum required.
I did a binary search for the boto3 version to see which one accepts the `max_instance_lifetime` param. I will experiment with botocore version. 
Thanks for sharing the helper method `boto3_at_least`.

Reply:
@tremble It requires botocore>=1.13.21